### ISSUE TITLE
Trailing stoploss at specific profit percentage

### ISSUE THREAD

**Issue by holycacao** [freqtrade/freqtrade#6164]
## Describe your environment

  * Operating system: Docker on Ubuntu 20.04.3
  * Python Version: Python 3.8.10
  * CCXT version: ccxt==1.50.30
  * Freqtrade Version: freqtrade 2021.12
  
## Your question
I've set the static stoploss to 2% and would like to make it a trailing stoploss once a trade has hit 1% profit. However, I can't seem to find the right config values to do so. The stoploss [docs](https://www.freqtrade.io/en/stable/stoploss/#trailing-stop-loss-only-once-the-trade-has-reached-a-certain-offset) state that `trailing_only_offset_is_reached` can be used with or without `trailing_stop_positive`, but I'm finding that to not be the case. From strategy:
```python
    stoploss = -0.02
    trailing_stop = True
    trailing_only_offset_is_reached = True
    trailing_stop_positive_offset = 0.01
```
When I run this, I get the following error:
```
2022-01-04 13:58:47,410 - 2022-01-04 14:22:58,832 - freqtrade.commands.trade_commands - ERROR - The config trailing_only_offset_is_reached needs trailing_stop_positive_offset to be more than 0 in your config.
```
If I set `trailing_stop_positive` to 0.02, I get this different error:
```
2022-01-04 14:22:58,832 - freqtrade.commands.trade_commands - ERROR - The config trailing_stop_positive_offset needs to be greater than trailing_stop_positive in your config.
```
I'm wondering if this is a bug in the config validation? Should line [128](https://github.com/freqtrade/freqtrade/blob/stable/freqtrade/configuration/config_validation.py#L128) in config_validation.py be checking the offset and not the tsl_positive config? Like this:
```python
    tsl_positive = float(conf.get('trailing_stop_positive', 0))
    tsl_offset = float(conf.get('trailing_stop_positive_offset', 0))
    tsl_only_offset = conf.get('trailing_only_offset_is_reached', False)

    if tsl_only_offset:
        # fix: check tsl_offset, not tsl_positive
        if tsl_offset == 0.0:
```

**Comment by xmatthias** [comment]
What your initial configuration would assume is that once you reach 1% profit, you'll trail with 2% (so the next stoploss is at -1%, still allowing for a 1% loss).

As this is usually not what you want, freqtrade has protections for it in place.
if you only trail when reaching a certain profit (`trailing_only_offset_is_reached`), you'll usually want that stoploss to be at least at break even, not trailing - but still allowing the trade to become negative.

**Comment by holycacao** [comment]
Okay, the -1% stoploss is what I intended so I thought maybe it was a bug since the docs seemed to indicate my config was allowed. I agree that generally you'd want to at least break even after any positive gain. I'm very new to this and am still doing a lot of tinkering. Thanks!
